karma.config.js

```diff
@@ -57,14 +58,14 @@ module.exports = function(config) {
     // strip templates of leading path
     ngHtml2JsPreprocessor: {
       moduleName: 'h.templates',
-      stripPrefix: 'h/templates/client/'
+      cacheIdFromPath: path.basename
     },
 
     // preprocess matching files before serving them to the browser
     // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors: {
       '**/*.coffee': ['browserify'],
-      'h/templates/client/*.html': ['ng-html2js'],
+      '../../templates/client/*.html': ['ng-html2js'],
```

Comment:
Next up maybe we should move the client-side templates down into these hierarchies, too, and maybe use something like https://www.npmjs.com/package/browserify-ng-html2js